src/userDashboard/userGraphs/UserBarGraph.js

```diff
@@ -0,0 +1,101 @@
+import { ResponsiveBar } from '@nivo/bar'
+import {BAR_GRAPH_KEYS} from '../dataFormat/ConstantsUserDashboard';
+
+export default function UserBarGraph ({ data }) {

```

Comment:
UserBarGraph? 
User does not say anything here.

Reply:
ReadingAndExercisesTimeGraph or something along those lines would be probably more explanatory.

Reply:
renamed both graphs to TranslatedWordsGraph and ReadingAndExercisesTimeGraph.